oak/server/rust/oak_loader/src/main.rs

```diff
@@ -0,0 +1,80 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! An utility binary to run Oak Runtime.
+//!
+//! To invoke, run the following command from the root of the repository:
+//!
+//! ```
+//! cargo run --package=oak_loader -- --application=<APP_CONFIG_PATH>
+//! ```
+
+use log::{error, info};
+use oak_runtime::{configure_and_run, proto::ApplicationConfiguration};
+use prost::Message;
+use std::{fs::File, io::Read, thread::sleep, time::Duration};
+use structopt::StructOpt;
+
+#[derive(StructOpt, Clone)]
+#[structopt(about = "Oak Loader")]
+pub struct Opt {
+    #[structopt(short, long, help = "Application configuration file")]
+    application: String,
+    // TODO(#806): Make agruments non-optional once TLS support is enabled.
+    #[structopt(long, help = "Path to the PEM-encoded CA root certificate")]
+    ca_cert: Option<String>,
+    #[structopt(long, help = "Path to the private key")]
+    private_key: Option<String>,
+    #[structopt(long, help = "Path to the PEM-encoded certificate chain")]
+    cert_chain: Option<String>,
+}
+
+fn read_file(filename: &str) -> Result<Vec<u8>, std::io::Error> {
+    let mut file = File::open(filename).map_err(|error| {
+        error!("Couldn't open file \"{}\": {:?}", filename, error);
```

Comment:
Does the `std::io::Error` not contain any information at all?

Reply:
It doesn't contain filename, so it just says: couldn't open a file